.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - 'iojs'
-  - '0.12'
-  - '0.10'
+  - '8'
+  - '6'
+  - '4'
```

Comment:
[clinton](https://github.com/samverschueren/clinton) detects this and will do it for you depending on the `engines.node` field :). It will also fix the `.travis.yml` file.